src/atama/receiver/node/receiver_node.cpp

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2021 Ichiro ITS
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "atama/receiver/node/receiver_node.hpp"
+#include "kansei_interfaces/msg/orientation.hpp"
+#include "ninshiki_interfaces/msg/detected_object.hpp"
+#include "ninshiki_interfaces/msg/detected_objects.hpp"
+#include "rclcpp/rclcpp.hpp"
+#include "tachimawari_interfaces/srv/get_joints.hpp"
+
+using namespace std::chrono_literals;
+
+namespace atama::receiver
+{
+
+ReceiverNode::ReceiverNode(rclcpp::Node::SharedPtr node, std::shared_ptr<atama::head::Head> head)
+: node(node), head(head)
+{
+  using kansei_interfaces::msg::Orientation;
+  using ninshiki_interfaces::msg::DetectedObject;
+  using ninshiki_interfaces::msg::DetectedObjects;
+  using tachimawari_interfaces::srv::GetJoints;
```

Comment:
most of these already declared here
https://github.com/ichiro-its/atama/blob/fdd5eeceff0fc9eb8131e6839c22fd8620d3f4ca/include/atama/receiver/node/receiver_node.hpp#L36-L38
you could use that instead

Reply:
I forgot I already declare it in the header. Hehehehe